tools/data-mapper/Microsoft.Health.Tools.DataMapper/ClientApp/src/services/Utils.ts

```diff
@@ -30,22 +103,168 @@ export const generateDeviceTemplate = (mappings: Mapping[]) => {
     return JSON.stringify(deviceTemplates, null, 4);
 }
 
+export const generateFhirMappings = (fhirTemplates: string): Promise<Mapping[]> => {
```

Comment:
Didn't go line-by-line, just curious whether we can modularize code in more functions/methods.